recipe/thirdparty/tpmodels/google.go

```diff
@@ -0,0 +1,21 @@
+package tpmodels
+
+import (
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+type TypeGoogleInput struct {
+	Config   []GoogleConfig
+	Override func(provider *GoogleProvider) *GoogleProvider
+}
+
+type GoogleConfig struct {
+	ClientID     string
+	ClientSecret string
+	Scope        []string
+}
```

Comment:
**why have you removed AuthorisationRedirect param?**
we can override the GetAuthURL on the provider to add or remove any param being passed.

**what if the user wants to pass custom params in the request body when it's exchanging auth code for access token? There used to be a config for that - now it's not there. why?**
it was not part of the provider config, it was being returned as part of the ProviderGetResponse. I'm not sure if there was an easy way to override that. This can also be achieved by overriding ExchangeAuthCodeForOAuthTokens function on the provider.

**Please see all the configs that we used to have and reevaluate which ones to keep and which ones to remove**
we have removed only AuthorisationRedirect.Params and IsDefault. None of the other config is removed.